crates/typst-library/src/layout/grid/mod.rs

```diff
@@ -143,19 +147,25 @@ pub struct GridElem {
     /// with that many `{auto}`-sized columns. Note that opposed to rows and
     /// gutters, providing a single track size will only ever create a single
     /// column.
+    ///
+    /// See the [track size section](#track-size) above for more details.
     pub columns: TrackSizings,
 
     /// The row sizes.
     ///
     /// If there are more cells than fit the defined rows, the last row is
     /// repeated until there are no more cells.
+    ///
+    /// See the [track size section](#track-size) above for more details.
     pub rows: TrackSizings,
 
     /// The gaps between rows and columns.
     ///
     /// If there are more gutters than defined sizes, the last gutter is
     /// repeated.
     ///
+    /// See the [track size section](#track-size) above for more details.
+    ///
     /// This is a shorthand to set `column-gutter` and `row-gutter` to the same
     /// value.
```

Comment:
I feel like those two paragraphs / blocks should be swapped

Reply:
I'm also struggling with it. The track size section provides details on _If there are more gutters than defined sizes, the last gutter is repeated_, so I think it should be put directly after that sentence.

But if you insist on swapping them, I'll do so.